src/balloon.scss

```
@@ -17,6 +17,7 @@
 
 :root {
   --balloon-color: #{$balloon-bg};
+  --balloon-foreground-color: #fff;
```

For ease of use, can you please just swich the name to `--balloon-text-color`?

```suggestion
  --balloon-text-color: #fff;
```